operations/helm/ct.yaml

```diff
@@ -4,7 +4,7 @@ target-branch: main
 chart-dirs:
   - operations/helm/charts
 chart-repos:
-  - minio=https://helm.min.io
+  - minio=https://charts.min.io/
```

Comment:
See related: https://github.com/grafana/helm-charts/issues/1534  .

Reply:
I see the @grafana/helm-charts#1534 issue but I still do not really get it. Should we parse the dependency from https://github.com/grafana/helm-charts/blob/main/.github/workflows/update-helm-repo.yaml#L131 or from https://github.com/grafana/helm-charts/blob/main/ct.yaml?

Reply:
Let me make a PR to add this new dependency and we can solve https://github.com/grafana/helm-charts/issues/1534 separately.